packages/webpack-config/src/plugins/ExpoDefinePlugin.ts

```diff
@@ -58,7 +58,7 @@ export function createClientEnvironment(
   const environment = getMode({ mode });
   const __DEV__ = environment !== 'production';
 
-  const ENV_VAR_REGEX = /^(EXPO_|REACT_NATIVE_|CI$)/i;
+  const ENV_VAR_REGEX = /^(CI$)/i;
```

Comment:
how do we ensure that this webpack-config change will only be rolled out with sdk 41?